src/gather_stats.ts

```diff
@@ -106,11 +120,8 @@ async function retrieve_nodes_from_network() {
             headers: { authorization: `token ${process.env.SECRET_GITHUB_PERSONAL_ACCESS_TOKEN}` },
         });
 
-        const rate_limit = {
-            spent: null as null | number,
-            remaining: null as null | number,
-            limit: null as null | number,
-        };
+        let spent: number;
+        let limit: number;
 
         {

```

Comment:
(Pre-existing) I know Javascript does not have scope by default, so variable declared in scope can be accessed from anywhere in a function after or before the scope. I hope TypeScript respects scope.

Reply:
Aha, I see, with `let` or `const` the block scope is respected!

Reply:
Yep! The Status Chart has always avoided `var` for this reason.